diff --git a/js/ui.js b/js/ui.js
index 78e3ecfbef000bd4b89f9356dacd07e30ff29477..6c97e29800177999496932962d316ef4a2d1fe17 100644
--- a/js/ui.js
+++ b/js/ui.js
@@ -691,51 +691,51 @@
         return `${condPrefix}致死耐え(戦1)`;
       case "firstHitCrit":
         return `${condPrefix}初撃会心`;
       case "overhealBarrierCap":
         return hasV
           ? `${condPrefix}余剰回復盾(上限${v}%)`
           : `${condPrefix}余剰回復盾`;
 
       case "magicPower":
         return hasV
           ? `${condPrefix}魔法攻撃力${sign}${v}`
           : `${condPrefix}魔法攻撃力`;
       case "healPower":
         return hasV ? `${condPrefix}回復力${sign}${v}` : `${condPrefix}回復力`;
       case "healReceived":
         return hasV ? `${condPrefix}回復量${sign}${v}%` : `${condPrefix}回復量`;
       case "damageReduction":
         return hasV
           ? `${condPrefix}被ダメージ-${Math.abs(v)}%`
           : `${condPrefix}被ダメージ軽減`;
       case "critDamage":
         return hasV
           ? `${condPrefix}クリダメ${sign}${v}%`
           : `${condPrefix}クリダメ`;
       case "lifeSteal":
-        return hasV ? `${condPrefix}吸血${sign}${v}%` : `${condPrefix}吸血`;
+        return hasV ? `${condPrefix}HP吸収${sign}${v}%` : `${condPrefix}HP吸収`;
       case "regen":
         return hasV ? `${condPrefix}再生${sign}${v}%` : `${condPrefix}再生`;
       case "hitHeal":
         return hasV
           ? `${condPrefix}攻撃時HP回復${sign}${v}`
           : `${condPrefix}攻撃時HP回復`;
       case "counterChance":
         return hasV ? `${condPrefix}反撃率${sign}${v}%` : `${condPrefix}反撃率`;
       case "counterDamage":
         return hasV
           ? `${condPrefix}反撃威力${sign}${v}%`
           : `${condPrefix}反撃威力`;
       case "desperationDamage":
         return hasV ? `${condPrefix}背水${sign}${v}%` : `${condPrefix}背水`;
       case "executeDamage":
         return hasV
           ? `${condPrefix}追い打ち${sign}${v}%`
           : `${condPrefix}追い打ち`;
       case "evadeHeal":
         return hasV
           ? `${condPrefix}回避回復${sign}${v}`
           : `${condPrefix}回避回復`;
       case "ailmentDurationDown":
         return hasV
           ? `${condPrefix}状態異常短縮${sign}${v}%`
